src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -67,13 +67,19 @@ public function generateSiteConfigFiles() {
 
       $copy_map = [
         $default_project_default_settings_file => $project_default_settings_file,
-        $project_default_settings_file => $project_settings_file,
         $blt_local_settings_file => $default_local_settings_file,
         $default_local_settings_file => $project_local_settings_file,
         $blt_local_drush_file => $default_local_drush_file,
         $default_local_drush_file => $project_local_drush_file,
       ];
 
+      // Only add the settings file if the default exists.
+      if (file_exists($project_default_settings_file)) {
+        $copy_map[] = [
```

Comment:
I think you want $copy_map[$project_default_settings_file] = $project_settings_file;